modules/jdbc/src/integrationTest/java/org/apache/ignite/jdbc/ItJdbcInsertStatementSelfTest.java

```diff
@@ -191,12 +191,13 @@ public void testPreparedExecute() throws Exception {
      * @throws Exception If failed.
      */
     @Test
+    @Disabled("IGNITE-20149")
```

Comment:
There is no need to disable this test, we just need to change the message 

> JdbcTestUtils.assertThrowsSqlException("PK unique constraint is violated", () -> stmt.execute(SQL));

Reply:
agree